src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -0,0 +1,161 @@
+package org.broadinstitute.hellbender.utils.fermi;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public final class FermiLiteAssembler {
```

Comment:
This class and public methods should probably have some minimal javadoc explaining them. (and the fact that they're backed by native code and may fail to load.  )